app/views/admin.njk

```diff
@@ -7,7 +7,7 @@
   {% include "includes/title-subtitle.njk" %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-half">
-    {% if authorisation===false or userRole==='Data Admin'%}
```

Comment:
Similat to the above really, and I know this is not new code, but it would be great if we could reduce the number of places the string is hard coded. This might be better done in the controller?